src/redux/Store.js

```diff
@@ -0,0 +1,19 @@
+import { createStore, applyMiddleware, compose } from "redux";
```

Comment:
Please change the "Store.js" to "store.js" since in index.js file we have it lower case.. so I wasn't able to run it locally because of this error.

Reply:
Changed it! It's weird, when I pulled from master it became uppercase, even though I had the same issue before and already fixed it lowercase, it was somehow back.